malasakit-django/pcari/views.py

```diff
@@ -110,8 +107,14 @@ def make_quantitative_question_ratings(respondent, responses):
 def make_comments(respondent, responses):
     for question_id, message in responses.get('comments', {}).iteritems():
         question = QualitativeQuestion(id=int(question_id))
-        yield Comment(respondent=respondent, question=question,
-                      language=respondent.language, message=message)
+
+        # Replaces empty messages with None so they can show up as placeholders in admin 
+        if message == "" or message.isspace():
+            yield Comment(respondent=respondent, question=question,
+                          language=respondent.language, message=None)
+        else:
+            yield Comment(respondent=respondent, question=question,
+                          language=respondent.language, message=message)
```

Comment:
Yeah, I saw that and it makes sense, but the issue is that I am trying to get [this](https://docs.djangoproject.com/en/1.10/ref/contrib/admin/#django.contrib.admin.ModelAdmin.empty_value_display) functionality to work by replacing empty strings with `"--empty response--"`. However, despite the documentation saying it replaces empty strings, it seems to only replace messages that are NULL. When I set all empty messages to be `" "` nothing shows up, which makes the comments not clickable, like with respondent 1613 in [this](https://puu.sh/wjiVt/095fdd8254.png) case. Not sure how to fix this issue.